pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -652,6 +661,34 @@ func (r *ReconcileLogStorage) getKibanaService(ctx context.Context) (*corev1.Ser
 	return &svc, nil
 }
 
+func (r *ReconcileLogStorage) getOIDCUserConfigMapAvailable(ctx context.Context) *corev1.ConfigMap {
```

Comment:
I think the name should just be `getOIDCUserConfigMap`.
Also I think it should return an error. We need to be able to tell if the error was because the resource doesn't exist or there was an error with the query.